### ISSUE TITLE
NTR: Modifier: eccrine sweat gland or duct localization to rash (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8039]
New term request Modifier: eccrine sweat gland or duct localization to rash (segal_220322184355):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000537
- label: Ecrrine sweat gland predominance
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: PMID:29489179
- def: Applies to an abnormality of the skin that is located in or surrounding eccrine sweat glands.
- comment: Eccrine sweat glands are simple, coiled, tubular glands present throughout the body, most numerously on the soles of the feet. Eccrine sweat glands serve a thermoregulatory function via evaporative heat loss. When the internal temperature of the body rises, sweat glands release water to the skin surface. There, it quickly evaporates, subsequently cooling the skin and blood beneath.; this is the most effective means of thermoregulation in humans. Eccrine sweat glands also participate in ion and nitrogenous waste excretion.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8039
- synonyms:
